platforms/nodejs/Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM buildpack-deps:stretch
+ARG DEBIAN_FLAVOR
+FROM buildpack-deps:${DEBIAN_FLAVOR}
+ARG DEBIAN_FLAVOR
+ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
+        make \
```

Comment:
I think we already have 'make' tool in nodejs images while I tested, we can still add here to make sure.

Reply:
this is ubuntu image .. the image you tested comes from a different image base .. so it was throwing error as it didn't have make .. i had to install it